libs/platform/auth/src/sapi/feature.ts

```diff
@@ -92,7 +109,9 @@ export class SapiAuthFeature extends OauthFeature implements AppFeature {
 
 export interface SapiAuthFeatureConfig extends OauthFeatureConfig {
   skipRoutes?: boolean;
+  /** deprecated since 1.4 */
```

Comment:
Maybe let's add a note that `guestIdentityInterceptor` should be used instead, so it's clear without taking a look into this interface.